client/src/components/v2/CameraSettings.js

```diff
@@ -32,25 +43,55 @@ export default function CameraSettings({ device }) {
     }
   }, [selectedOverlay, setActiveOverlay]);
 
-  return (
-    <Card>
-      <Card.Body>
-        <Card.Title>{`${name} display`}</Card.Title>
-        {controls ? (
-          <RadioSelector
-            options={controls.overlays}
-            value={selectedOverlay}
-            onChange={setSelectedOverlay}
-          />
-        ) : (
-          <Card.Subtitle>Waiting for response...</Card.Subtitle>
-        )}
-      </Card.Body>
-      <Card.Footer>
-        <Card.Link href="#" onClick={handleSave}>Save</Card.Link>
-      </Card.Footer>
-    </Card>
-  );
+  const handleMessageSubmit = useCallback(() => {
+    sendMessage(message.current.value);
+    message.current.value = '';
+  }, []);
+
+  return ([
+    <div>
+      <Card>
+        <Card.Body>
+          <Card.Title>{`${name} display`}</Card.Title>
+          {controls ? (
+            <RadioSelector
+              options={controls.overlays}
+              value={selectedOverlay}
+              onChange={setSelectedOverlay}
+            />
+          ) : (
+            <Card.Subtitle>Waiting for response...</Card.Subtitle>
+          )}
+        </Card.Body>
+        <Card.Footer>
+          <Card.Link href="#" onClick={handleSave}>Save</Card.Link>
+        </Card.Footer>
+      </Card>
+    </div>,
+    <div>
+      <Card>
+        <Card.Body>
+          <Card.Title>{`${name} Message`}</Card.Title>
+          <InputGroup className="mt-3">
+            <FormControl
+              ref={message}
+              placeholder="Type Message Here"
+            />
```

Comment:
It would be nice if the message sent when we pressed enter. Had a quick google, this looks like it'd work https://stackoverflow.com/a/44945013

Reply:
Thanks, will implement that.